quay_search.py

```diff
@@ -0,0 +1,131 @@
+#! /home/wolffj/miniconda2/bin/python
```

Comment:
Yes :) a little bit more background: http://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash